Sources/Core/GTMSessionFetcher.m

```diff
@@ -1692,14 +1710,14 @@ - (void)authorizeRequest {
   }
 }
 
-- (void)authorizer:(id<GTMFetcherAuthorizationProtocol>)auth
-              request:(NSMutableURLRequest *)authorizedRequest
-    finishedWithError:(NSError *)error {
+- (void)authorizer:(id)auth
```

Comment:
Shouldn't this be nullable also since you use a weak ivar to pass it?

If you do that, you could even make the block always pass nil and just document that we don't use this here any more and it is only there to the selector based signature.